lib/enum/enum.rb

```diff
@@ -14,6 +14,17 @@ def enum_accessor(*field_names)
         reload
       end
 
+      def enum(columns)
+        method_definitions = []
+        columns.each do |column, values|
+          values = values.zip(values.map(&:to_s)).to_h if values.is_a?(Array)
+          values.each do |method, value|
+            method_definitions << "def #{method}?; #{column} == '#{value}'; end"
```

Comment:
```suggestion
            method_definitions << "def #{method}?; #{column} == #{value.inspect}; end"
```

Will `value.inspect` work?
I'd prefer to use that if it buys us enumeration with columns that contain integers.

Strings are most important, integers are a nice to have, and symbols are a very distant nice to have. I think we get all 3 with inspect

Reply:
I have decided to go with your suggestion!
https://github.com/active-hash/active_hash/pull/321/commits/5cab8ac5e9d83af11093997e1cb24955a1a999ad

```ruby
bundle exec rspec spec/enum/enum_spec.rb   
...........

Finished in 0.01806 seconds (files took 0.49792 seconds to load)
11 examples, 0 failures
```